runners/core-construction-java/src/test/java/org/apache/beam/runners/core/construction/ValidateRunnerXlangTest.java

```diff
@@ -177,6 +232,14 @@ public void coGroupByKeyTest() {
     PAssert.that(col).containsInAnyOrder("0:1,2,4", "1:3,5,6");
   }
 
+  /**
+   * Motivation behind combineGloballyTest.
```

Comment:
Let's keep these unrelated documentation updates in a separate PR.

Reply:
Resolving since this was caused due to botched git workflows and have since been corrected.